apps/andi/lib/andi/input_schemas/ingestions/extract_steps/extract_auth_step.ex

```diff
@@ -86,8 +86,17 @@ defmodule Andi.InputSchemas.Ingestions.ExtractAuthStep do
 
   defp validate_body_format(%{changes: %{body: body}} = changeset) do
     case Jason.decode(body) do
-      {:ok, _} -> changeset
-      {:error, _} -> Changeset.add_error(changeset, :body, "could not parse json", validation: :format)
+      {:ok, _} ->
+        changeset
+
+      {:error, _} ->
+        try do
+          SweetXml.parse(body)
+          changeset
+        catch
+          :exit, _ ->
+            Changeset.add_error(changeset, :body, "could not parse json", validation: :format)
```

Comment:
We should probably add a blurb about not being able to parse xml as well

Reply:
This isn't what the UI shows, due to the error_tag function, but still good catch